js/ui/windowManager.js

```diff
@@ -565,14 +553,7 @@ var WindowManager = class WindowManager {
     }
 
     onSettingsChanged(settings, key, type) {
-        switch (settings.schema) {
-            case 'org.cinnamon':
-                this.settingsState[key] = global.settings[type](key);
-                break;
-            case 'org.cinnamon.muffin':
-                this.settingsState[key] = this.settings[type](key);
-                break;
-        }
+        this.settingsState[key] = global.settings[type](key);
```

Comment:
Since bracket notation is faster, I think the previous approach would be preferred where we dynamically look up properties for all settings. It's a little extra code, but I don't think it's hard to follow here